### PR TITLE
Create PlantFiche and replace list with grid

### PR DESCRIPTION
**Pull request by anna-wro**
Kapture 2020-12-12 at 20 48 27

Screenshot 2020-12-12 at 21 12 30

### CODE REVIEW

File: src/components/PlantFiche/PlantFiche.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+type PlantFicheProps = Readonly<{
+  plant: PlantType;
+}>;
+
+const PHOTOS_TO_DISPLAY = 1;
+
+export default function Plant({ plant }: PlantFicheProps) {
+  const thumbnail = plant.thumbnailID ?  useThumbnail(plant.thumbnailID) : null;
+  const url = thumbnail ? getPhotoUrl(thumbnail) : null;
```
Comment (anna-wro) [inline src/components/PlantFiche/PlantFiche.tsx:26]: this will be refactored later (this or next PR depending when you review it ;)

**Comment by anna-wro** [comment]
@coderitual I'm not touching this PR anymore, search will be in another one

**Review by coderitual — APPROVED** [review]
Nice job @anna-wro! let me know if you want to discuss anything from this PR on our meeting.

Side note: Not sure if this image blinking happening only on local dev or gif itslef but if so, I think it should be on our radar to fix :)

**Comment by anna-wro** [comment]
@coderitual yeah, let's fix it next time, and maybe handle loading images only when the fiche is visible, not 60+ of them :). 
No more questions so far.

**Comment by anna-wro** [comment]
just typescript, as always ;)

**Comment by coderitual** [comment]
yea, any concerns around typescript we can resolve on review as well :)

**Pull request merged**